### PR TITLE
Increasing the codecov for audioplayer multistream (sdkv2 mainbranch)

### PR DESCRIPTION
**Pull request by ig-perez**
Adding shuffle, loop and getting to the beginning and end of the playlist.

### CODE REVIEW

File: multiple-streams/test/intents.yml [hunk 1]
```diff
@@ -92,4 +92,50 @@ configuration:
...
+- test: "Sequence 07. Test scenario: Handles AMAZON.ShuffleOnIntent."
+- LaunchRequest: "*"
+- "begin podcast": "*"
+- AMAZON.ShuffleOnIntent:
+  - intent: AMAZON.ShuffleOnIntent
+  - response.outputSpeech.ssml: "This is Episode"
+
+---
+- test: "Sequence 08. Test scenario: Handles AMAZON.ShuffleOffIntent."
+- LaunchRequest: "*"
```
Comment (jkelvie) [inline multiple-streams/test/intents.yml]: You can just say: 
`- LaunchRequest`

File: multiple-streams/test/intents.yml [hunk 2]
```diff
@@ -92,4 +92,50 @@ configuration:
...
   - response.outputSpeech.ssml: "You have reached the end of the playlist"
   - response.shouldEndSession: undefined
-  - response.directives[0].type: "AudioPlayer.Stop"
\ No newline at end of file
+  - response.directives[0].type: "AudioPlayer.Stop"
+
+---
+- test: "Sequence 07. Test scenario: Handles AMAZON.ShuffleOnIntent."
+- LaunchRequest: "*"
+- "begin podcast": "*"
```
Comment (jkelvie) [inline multiple-streams/test/intents.yml]: `- "begin podcast"`

File: multiple-streams/test/intents.yml [hunk 3]
```diff
@@ -92,4 +92,50 @@ configuration:
...
+  - intent: AMAZON.LoopOnIntent
+  - response.outputSpeech.ssml: "/.*sorry, this is not a valid command.*/i"
+
+---
+- test: "Sequence 10. Test scenario: Handles AMAZON.LoopOffIntent."
+- LaunchRequest: "*"
+- "begin podcast": "*"
+- AMAZON.LoopOffIntent:
+  - intent: AMAZON.LoopOffIntent
+  - response.outputSpeech.ssml: "/.*sorry, this is not a valid command.*/i"
```
Comment (jkelvie) [inline multiple-streams/test/intents.yml]: Do not use regular expressions unless necessary. When using regexes, no need to quote

**Comment by ig-perez** [comment]
Just pushed changes based on your comments

**Pull request merged**